site/content/docs/5.3/content/tables.md

```diff
@@ -36,6 +36,15 @@ See [more information about the tables structures](https://a11y-guidelines.orang
 <!-- Boosted mod: only .table-dark is allowed -->
 Use contextual class to color tables, table rows or individual cells.
```

Comment:
This sentence should be in the `<details><summary>` and we should add back the information callout from Bootstrap too within this `<details><summary>`